### DISCUSSION TITLE
Can't set custom colors

### DISCUSSION THREAD

**Discussion by waikoo** [kyazdani42/nvim-web-devicons#278] [Q&A]
Can I get some help here please?

I have nvim-web-devicons installed via packer. In nvim/after/plugin/web-devicon.lua I have the following content:

```lua
require("nvim-web-devicons").set_icon {
  css = {
    color = "blue",
  }
}
```
What else do I need to do to set the color?
It should work with fern, right? I have fern using this plugin via:

`vim.g["fern#renderer"] = "nvim-web-devicons"`

**Comment by alex-courtis** [comment]
You're missing the rest of the icon definition.

Try
```lua
  ["css"] = {
    icon = "",
    color = "Blue",
    cterm_color = "75",
    name = "Css",
  },
```

See `:help gui-colors` for available colours.
